_config.yml

```
@@ -481,6 +482,17 @@ apps:
     -   anat
     description: a BIDS app for pseudo-anonymization of neuroimaging data
 
+-   gh: PennLINC/xcp_d
+    dh: pennlinc/xcp_d
+    description: Post-processing of fMRIPrep for functional connectivity analysis
```

```suggestion
    description: Post-processing of fMRIPrep, nibabies, HCP, and DCAN BOLD derivatives for functional connectivity analysis
```

fMRIPrep and nibabies are the only first-order preprocessing pipelines XCP-D supports, so I'd be fine with just "fMRIPrep and nibabies" if you think that makes more sense.

---

Na your suggestion is good with me.